apps/data/database/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM postgres:16-alpine
 
 COPY ./postgresql.conf /etc/postgresql.conf
 
+COPY recommender/trainingData.csv /docker-entrypoint-initdb.d/recommender/
```

Comment:
Why is it both copying `trainingData.csv` into the dockerfile and creating a volume? Shouldn't we do one or the other?